lib/game.rb

```diff
@@ -0,0 +1,240 @@
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+
+
+
+class Game
+
+    attr_reader :player1,
+                :player2,
+                :deck1,
+                :deck2
+
+    def initialize 
+
+        
+
+        create_decks
+
+        
+        @player1 = Player.new("Mufasa", @deck1)
+        @player2 = Player.new("Skar", @deck2)
+        
+
+    end
+
+
+
+
+
+    def create_high_value(suits, suit)
+
+        suits << Card.new(suit, "Jack", 11)
+        suits << Card.new(suit, "Queen", 12)
+        suits << Card.new(suit, "King", 13)
+        suits << Card.new(suit, "Ace", 14)
+
+    end
+
+
+
+
+
+
+    def create_decks
+
+        rank_values = [2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+        diamonds = []
+        hearts = []
+        spades = []
+        clubs = []
+
+        full_deck_cards = []
+
+
+        rank_values.each do |rank_value|
+
+            diamonds << Card.new(:diamond, "#{rank_value}", rank_value)
+
+            hearts << Card.new(:heart, "#{rank_value}", rank_value)
+
+            spades << Card.new(:spade, "#{rank_value}", rank_value)
+
+            clubs << Card.new(:club, "#{rank_value}", rank_value)
+            
+        end
+
+
+        create_high_value(diamonds, :diamond)
+        create_high_value(hearts, :heart)
+        create_high_value(spades, :spade)
+        create_high_value(clubs, :club)
+
+
+        full_deck_cards = diamonds + hearts + spades + clubs
+        full_deck_cards = full_deck_cards.shuffle
```

Comment:
I like the shuffle. Without a shuffle we end up playing the same game every time.